frontend/src/views/WheresMyGene/components/ScreenTint/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { EXCLUDE_IN_SCREENSHOT_CLASS_NAME } from "../GeneSearchBar/components/SaveImage";
+import Loader from "../Loader";
+import { StyledDiv } from "./style";
+
+export default function ScreenTint(): JSX.Element {
+  return (
+    <>
+      <Loader />
+      <StyledDiv className={EXCLUDE_IN_SCREENSHOT_CLASS_NAME}>
+
+      </StyledDiv>
```

Comment:
self closing tag will do!

```suggestion
      <StyledDiv className={EXCLUDE_IN_SCREENSHOT_CLASS_NAME} />
```